rosidl_runtime_py/get_interfaces.py

```diff
@@ -171,12 +171,16 @@ def get_interface_path(interface_name: str) -> str:
             f"Invalid name '{interface_name}'. Expected at least two parts separated by '/'")
     if not all(parts):
         raise ValueError(f"Invalid name '{interface_name}'. Must not contain empty parts")
+    if '..' in parts:
+        raise ValueError(f"Must not contain '..'")
     # By convention we expect the first part to be the package name
     prefix_path = has_resource('packages', parts[0])
     if not prefix_path:
         raise LookupError(f"Unknown package '{parts[0]}'")
 
     interface_path = os.path.join(prefix_path, 'share', interface_name)
+    if not parts[-2] in ['msg', 'srv', 'action']:
```

Comment:
While the namespaces at the moment only use on of these three values the naming scheme does intentionally support arbitrary namespace to allow adding things like e.g. `constant` or `param` in the future.

Reply:
In my opinion, the input of `ros interface` may not only come from current user. For example we can develop a web UI and have the result of `ros interface` to show on the web for other users. In this case malicious user might have the wrong input to get the content of any arbitrary file and this is not what we expect. Developers know `cat` will show the content of files, but may not know `ros interface` do the same thing.

Although this might be a special case, I prefer to limit the input and only show what we expect to avoid potential problem. However this is just my opinion, so it's fine to me to close the PR if you think this is not a issue.


> While the namespaces at the moment only use on of these three values the naming scheme does intentionally support arbitrary namespace to allow adding things like e.g. `constant` or `param` in the future.

OK, I'll remove the namespace checking.